RELEASE_NOTES

```diff
@@ -1,5 +1,9 @@
 			OPENARC RELEASE NOTES
 
+1.0.0.beta0     2018/08/05
```

Comment:
This isn't needed for the 1.0.0 release, which is going to contain a single "Initial release" entry.